pkgs/development/python-modules/hdbscan/default.nix

```diff
@@ -0,0 +1,32 @@
+
+{ lib
+, buildPythonPackage
+, cython
+, numpy
+, scipy
+, scikitlearn
+, pythonOlder
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "hdbscan";
+  version = "0.8.12";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0yxi34frg2jwyvjl942qy4gq5pbx8dq4pf4p28d1xah8njchfqir";
+  };
+
+  # No tests in archive
+  doCheck = false;
+   
+  propagatedBuildInputs = [ cython numpy scipy scikitlearn ];
+
+  meta = with lib; {
+    description = "HDBSCAN - Hierarchical Density-Based Spatial Clustering of Applications with Noise. HDBSCAN is a clustering algorithm with a scikit-learn compatible API; it doesn't require defining the number of clusters sought, and fares well with oddly shaped clusters.";
```

Comment:
Too long as well.

Reply:
Should have learned this by now, but its somehow counter-intuitive; fixed now.